src/components/url-input.yml

```diff
@@ -0,0 +1,15 @@
+tooltip_youtube:
+  en: Youtube URL
+  de: Youtube URL
+tooltip_internal-public:
+  en: Internal public storage URL
+  de: Interner öffentlicher Speicher URL
+tooltip_internal-private:
+  en: Internal private storage URL
+  de: Interner privater Speicher URL
+tooltip_external:
+  en: External URL
+  de: Externer URL
+tooltip_unsupported:
+  en: Unsupported value
+  de: Nicht unterstützten Wert
```

Comment:
Maybe better "Nicht unterstützte URL"?

Reply:
But it could be that the user did type something abstract like "abc". That's why I didn't want to call it "URL".